src/table.rs

```diff
@@ -65,31 +70,146 @@ pub struct Table {
     inner: Arc<TableInner>,
 }
 
+/// `AsRef<TableInner>` is only used in `init_biggest_and_smallest`
+/// to construct a table iterator from `&TableInner`.
+impl AsRef<TableInner> for TableInner {
+    fn as_ref(&self) -> &TableInner {
+        self
+    }
+}
+
 impl TableInner {
     /// Create an SST from bytes data generated with table builder
-    pub fn create(_path: &Path, _data: Bytes, _opt: Options) -> Result<TableInner> {
-        unimplemented!()
+    fn create(path: &Path, data: Bytes, opt: Options) -> Result<TableInner> {
```

Comment:
```suggestion
    fn create(path: &Path, data: Bytes, opts: Options) -> Result<TableInner> {
```
same for other signatures

Reply:
Alright, I'll try to make them consistent